lib/linguist/languages.yml

```diff
@@ -1291,6 +1291,16 @@ F#:
   codemirror_mode: mllike
   codemirror_mime_type: text/x-fsharp
   language_id: 105
+F*:
+  type: programming
+  color: "#572e30"
+  aliases:
+  - fstar
+  extensions:
+  - ".fst"
```

Comment:
@pchaigno There is also the `.fsti` extension, for interface files, which we use a lot. :)
(as OCaml has `.ml` and `.mli`)

Reply:
Should I be looking for different keywords for `.fsti` files? I [can't find a lot of them](https://github.com/search?q=extension%3Afsti+module+OR+assume+OR+let+OR+val&type=Code).

Reply:
No, there's no syntactic distinction between `.fst` and `.fsti` files (`.fsti`s are "partial" `.fst`s, in a way), so I believe you can just slap `.fsti` here and be done!

Reply:
@mtzguido Unfortunately, there doesn't seem to be enough F* `.fsti` files on GitHub to meet the in-the-wild usage constraint.